src/nimvscode/nimBuild.nim

```diff
@@ -135,18 +135,18 @@ proc parseErrors(lines: seq[cstring]): seq[CheckResult] =
     if line.startsWith("Hint:") or line == "" or dots.test(line):
       continue
 
-    let match = newRegExp(r"^([^(]*)?\((\d+)(,\s(\d+))?\)( (\w+):)? (.*)")
+    let match = newRegExp(r"^(>+ )?([^(]*)?\((\d+)(,\s(\d+))?\)( (\w+):)? (.*)")
```

Comment:
Should the preceding ">" be ignored/dropped from output?

Reply:
The leading '>' are just for improved readability in the terminal rendering.
Where are proposing it to be output?

Reply:
I thought the lines might have been grouped up so that character might still be useful. Not a big deal.

Did you test it?

Reply:
The hints are parsed out into individual diagnostic messages.
Here is a screenshot with this change:
https://github.com/nim-lang/Nim/pull/18418#issuecomment-874728129

I noticed there is a problem matcher regex in `package.json`. What is that for?

Reply:
Those are when the errors are in the console when you do a build, it'll output to screen, if there are warnings or dailies with file and line numbers it'll parse that and allow clock through to the workspace file.

Updating that if it's need would be handy.

Reply:
done